plugin/trino-password-authenticators/src/test/java/io/trino/plugin/password/ldap/LdapUtil.java

```diff
@@ -36,7 +36,8 @@
     public static final String MEMBER = "member";
 
     private LdapUtil()
-    {}
+    {
```

Comment:
Can we revert this change.

Reply:
yes, i will tidy up the formatting in this class in general actually.